app/src/lang/translations/zh-CN.yaml

```diff
@@ -825,7 +825,7 @@ page_help_collections_overview: '**模型总览** - 您可以访问的所有模
 page_help_collections_collection: >-
   **浏览条目** - 列出所有 {collection} 集合中你可以访问的条级数据。 可以通过设置自定义布局、过滤器和排序来调整视图，甚至将不同的设置保存为书签以便快速访问。
 page_help_collections_item: >-
-  **条目详情** - 查看和管理此条目的表单。侧边栏还包含完整的修改历史和评论。
+  **条目详情l** - 查看和管理此条目的表单。侧边栏还包含完整的修改历史和评论。
```

Comment:
@rijkvanzanten seems like this reverted the changes in #11767 as well... 🤔 the `1` was specifically removed in that PR (as it is a typo): https://github.com/directus/directus/pull/11767/files#diff-4f013a23849fcfc095e756e051b826871a1c6ddd0a77a8ca984dd35964b05fb6R828

Reply:
@azrikahar Translation changes from the repo must be synced to Crowdin to take effect, otherwise it will be overwritten. I guess we could set up an action for this, but generally I'd suggest to only edit translations directly via Crowdin. See https://support.crowdin.com/github-integration/#uploading-translations-from-repo

Reply:
ohh thanks for clarifying. Feel like there's a slight disconnect in the sense that we may just have tentative translations in a draft/open PR, but not necessarily will/should end up in the final translations compared to editing directly via Crowdin. But maybe this is still somewhat infrequent to warrant an action for now.

Reply:
You're very welcome! In which cases do you think this happens?
Btw. I've now added the changes from #11767 to Crowdin, to make sure those aren't lost.

Reply:
> In which cases do you think this happens?

The slight disconnect _might_ happen when let's say we add a new feature, so we'll need to add new translations for some new fields during development. However the PR may take a while to review so we can't add the new translations yet to be safe, since it the PR itself might get rejected (really a stretch here). Afterwards we'll need to remember to add them in once it's actually merged.

This is honestly really super nitpicky here... so let's just put a pin on it first 😄 all good!

> Btw. I've now added the changes from https://github.com/directus/directus/pull/11767 to Crowdin, to make sure those aren't lost.

Thank you!

Reply:
> > In which cases do you think this happens?
> 
> The slight disconnect _might_ happen when let's say we add a new feature, so we'll need to add new translations for some new fields during development. However the PR may take a while to review so we can't add the new translations yet to be safe, since it the PR itself might get rejected (really a stretch here). Afterwards we'll need to remember to add them in once it's actually merged.
> 
> This is honestly really super nitpicky here... so let's just put a pin on it first 😄 all good!
> 

Ah I see, thanks for clarifying 👍